conda_libmamba_solver/index.py

```diff
@@ -174,7 +174,7 @@ def _fetch_channel_url_with_conda(self, url):
         repo = self._repo_from_json_path(
             self._pool,
             url,
-            subdir_data.cache_path_json,
+            str(subdir_data.cache_path_json),
```

Comment:
That's an unfortunate change :-/

Reply:
Yeah :/ but luckily we will be changing how we depend on subdir_data soon.